packages/back-end/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "back-end",
   "version": "0.0.1",
   "private": true,
+  "engines": {
+    "node": ">=16"
+  },
```

Comment:
I believe we also need to update the `@types/node` dependency on line 89

Reply:
A quick update, it looks like many of the errors that I got were red herrings - I removed the node_modules, along with the yarn.lock file and rebuilt it and now everything is happy.